apps/stripe/src/app/api/saleor/payment-gateway-initialize-session/response-data-shape.ts

```diff
@@ -0,0 +1,9 @@
+import { z } from "zod";
+
+export const PaymentGatewayInitializeResponseDataShape = z.object({
+  stripePublishableKey: z.string(),
```

Comment:
Maybe its too much, but if we want extra security we can do refine() and run PublsihableKey.create here

Reply:
actually in use-case we take its value from StripeConfig VO which itself is needs to have proper values 🤔